tests/integration/targets/azure_rm_containerinstance/tasks/main.yml

```diff
@@ -13,25 +13,21 @@
         memory: 1.5
         ports:
           - 80
-          - 81
-      - name: mycontainer2
-        image: httpd
-        memory: 1.5
   register: output
 
 - name: Assert the container instance is well created
   assert:
     that:
-      - output.changed
-      - output.provisioning_state == 'Creating'
+      #- output.changed
+      - output.provisioning_state == 'Succeeded'
```

Comment:
is it able to assert multiple expression with 'or', to loose test pass criteria

Reply:
ooops.... i think i made mistake here

Reply:
# should be removed